src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -54,7 +54,7 @@ protected GameManagementService(RoomRepository repository, SocketService socketS
         currentGameMap = new HashMap<>();
     }
 
-    protected Game getGameFromRoomId(String roomId) {
+    public Game getGameFromRoomId(String roomId) {
```

Comment:
Why was this changed from `protected` to `public`?

Reply:
In order to do the socket stuff I had to call this method from the `WebSocketConnectionEvents.java` so I had to change this to public (I might actually be able to change this back to protected though reading one of your other comments/proposed changes, so we'll see haha)

Reply:
Just changed it back - made a separate method as suggested in another comment so this doesn't need to be public anymore